### ISSUE TITLE
Idea to enhance the addon ?

### ISSUE THREAD

**Issue by razeNFR** [mrcanelas/tmdb-addon#48]

**Comment by mrcanelas** [comment]
It got a little confusing, I couldn't understand what you suggested, but I'm limited by the TMDB API, for example in trends I can't filter by genre, the trends endpoint is only one.

**Comment by razeNFR** [comment]
> It got a little confusing, I couldn't understand what you suggested, but I'm limited by the TMDB API, for example in trends I can't filter by genre, the trends endpoint is only one.

Yeah sorry im rly bad in english, but adding "trend" and "new" category will be great already
